plugins/inputs/conntrack/conntrack_test.go

```diff
@@ -9,7 +10,9 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/influxdata/telegraf/plugins/inputs/system"
 	"github.com/influxdata/telegraf/testutil"
+	"github.com/shirou/gopsutil/v3/net"
 	"github.com/stretchr/testify/require"
```

Comment:
Can we please make this
```suggestion
	"github.com/shirou/gopsutil/v3/net"
	"github.com/stretchr/testify/require"

	"github.com/influxdata/telegraf/plugins/inputs/system"
	"github.com/influxdata/telegraf/testutil"
```